TASK/ClearPedTasks.md

```diff
@@ -10,5 +10,6 @@ void CLEAR_PED_TASKS(Ped ped);
 
 
 ## Parameters
-* **ped**: 
+* **ped**: Ped id. Use PlayerPedId() for your own character.
 
+Clear a ped's tasks. Stop animations and other tasks created by script.
```

Comment:
Please move this line above the Parameters section, otherwise it will not actually appear in the description.
[example](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md)